docs/hello-world.md

```diff
@@ -6,6 +6,15 @@ next: docs/development.md
 
 A Probot app is just a [Node.js module](https://nodejs.org/api/modules.html) that exports a function:
 
+To install node (Mac OS X)
+ * Install [homebrew](https://brew.sh/)
+ * `brew install node`
+
+After installing node:
+ * `npm install node`
```

Comment:
Why are we installing [`node`](http://npm.im/node) from npm?

Reply:
@j-f1 Oops, typo on my part, I meant to put `npm install probot` here. Fixing with a new commit.